src/api.tsx

```diff
@@ -0,0 +1,18 @@
+import { TodoType, UserType } from './types';
+
+const usersUrl = 'https://jsonplaceholder.typicode.com/users';
+const todosUrl = 'https://jsonplaceholder.typicode.com/todos';
```

Comment:
зачем нам две ссылки?
```suggestion
const API_URL = 'https://jsonplaceholder.typicode.com/';
```

в нужном месте добавляй `users / todos`

Reply:
Ок